packages/content-type-read-receipt/README.md

```diff
@@ -28,6 +35,10 @@ yarn add @xmtp/content-type-read-receipt
 pnpm i @xmtp/content-type-read-receipt
 ```
 
+## Provide an opt-out option
+
+While this is a per-app decision, the best practice is to provide users with the option to opt out of sending read receipts. If a user opts out, when they read a message, a read receipt will not be sent to the sender of the message.
+
 ## Create a read receipt
 
 With XMTP, read receipts are represented as objects with the following keys:
```

Comment:
i think we should update this to read: `With XMTP, read receipts are represented as empty objects.`

and then below, remove L#46.

and update L#49 - L#51 to be:
```
const readReceipt: ReadReceipt = {};
```

and also L#59 - L#64 to be:

```
await conversation.messages.send(
  {},
  ContentTypeReadReceipt,
);
```